topics/single-cell/metadata.yaml

```
@@ -3,6 +3,7 @@ name: single-cell
 type: use
 title: Single Cell
 summary: Training material and practicals for all kinds of single cell analysis (particularly scRNA-seq!). When you generate your lovely gene lists for your cells, consider checking out our Transcriptomics tutorials for further network analysis!
+What else do you want to see? You can submit tool and/or tutorial requests, as well as up-vote requests from others at our [Single Cell Community Planning site](https://github.com/orgs/galaxyproject/projects/31/views/1).
```

```suggestion
summary: |
  Training material and practicals for all kinds of single cell analysis (particularly scRNA-seq!). When you generate your lovely gene lists for your cells, consider checking out our Transcriptomics tutorials for further network analysis!
  
  What else do you want to see? You can submit tool and/or tutorial requests, as well as up-vote requests from others at our [Single Cell Community Planning site](https://github.com/orgs/galaxyproject/projects/31/views/1).
  {: .alert.alert-success}
```

currently like that it'd be invalid yaml (i guess the workflow will error, needs to be like this). 

I *think* just adding alert/alert-success is sufficient to get it in a box